packages/gatsby/src/utils/page-data.js

```diff
@@ -1,6 +1,7 @@
 const fs = require(`fs-extra`)
 const path = require(`path`)
 const { store } = require(`../redux`)
+const Promise = require(`bluebird`)
```

Comment:
Is there a particular need for this? Node 8+ supports native promises. It looks like no BlueBird specific features were used.

Reply:
Removed 🙇